src/jsifier.js

```diff
@@ -216,7 +220,7 @@ function ${name}(${args}) {
           } else if (VERBOSE || WARN_ON_UNDEFINED_SYMBOLS) {
             warn(msg);
           }
-          if (ident === 'main' && STANDALONE_WASM) {
+          if (undefinedSym === 'main' && STANDALONE_WASM) {
```

Comment:
I'd expect this line to check for either `main` or `main_argc_argv`... what am I missing?